app/controllers/api/applications_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class Api::ApplicationsController < Api::BaseController
+  before_action :find_service
+  before_action :activate_submenu
+
+  include ThreeScale::Search::Helpers
+  include DisplayViewPortion
+  helper DisplayViewPortion::Helper
+
+  activate_menu :main_menu => :serviceadmin, sidebar: :applications
+  sublayout 'api/service'
+
+  def index
+    # TODO: This code is REAAAAAAAAAAAALLY bad but it is copied and pasted from Buyers::ApplicationsController because
```

Comment:
we decided to do it this way for lack of time and to not take the risk to introduce bugs. Let's refactor later 😄 I will create an issue for this refactoring once this PR is merged so we don't forget about this 😄 
It was decided in:
![image](https://user-images.githubusercontent.com/11318903/46007774-2e1efc00-c0bb-11e8-951c-822c982d6ded.png)
and:
![image](https://user-images.githubusercontent.com/11318903/46007888-79d1a580-c0bb-11e8-8cbe-b8746fa2fade.png)